lib/launch/launch_configs.dart

```diff
@@ -38,6 +42,12 @@ class LaunchConfigurationManager implements Disposable, StateStorable {
     return _getCreateProjectConfig(path).getConfigs();
   }
 
+  List<LaunchConfiguration> getAllConfigs() {
```

Comment:
What's the naming convention for things like getters?
Should we be using this pattern or `List<LaunchConfigurations> get allConfigs {` ?